properties.ini

```diff
@@ -0,0 +1,170 @@
+; this file is used to define all necessary options for virt-who testing
+; all default values are null, need to update according to the test requirements
+
+[host]
+; the rhel system to run virt-who service
+; virtwho/docker.py and kickstart.py can help to install a host
+ip=
+user=
+passwd=
+
+[repo]
+; the base url of the rhel compose for special cases
+rhel_base=
+
+[proxy]
+; a proxy server is necessary if run the related test cases
+server=
+port=
+
+[rhsm]
+; the datas of stage or product candlepin server
+server=
+user=
+passwd=
+org=
+
+[satellite]
+; the datas of satellite server
+; virtwho/satellite.py can help to deploy a satellite server
+server=
+admin_user=
+admin_passwd=
+default_org=
+default_env=
+extra_org=
+activation_key=
+
+[vcenter]
+; esx_ datas are for the esxi host which the guest is based on
+vcenter_ip=
+vcenter_admin_user=
+vcenter_admin_passwd=
+vcenter_ssh_user=
+vcenter_ssh_password=
+esx_user=
+esx_passwd=
+guest_name=
+guest_user=
+guest_passwd=
+; hypervisor-builder can help to get below datas by searching guest name
+esx_ip=
+esx_uuid=
+esx_hwuuid=
+esx_hostname=
+esx_version=
+esx_state=
+esx_cpu=
+esx_cluster=
+guest_ip=
+guest_uuid=
+guest_state=
+
+[xen]
+xen_ip=
+xen_user=
+xen_passwd=
```

Comment:
The same as vcenter:
[xen]
server=
username=
password=
uuid=
hostname=
version=
state=
cpu=

The will be called by: xen.server, xen.username.....xen.uuid, xen.hostname

hyperv, libvirt, xen, kubevirt mostly should have the same options.
vcenter and rhevm are more closely

Reply:
done it